Dockerfile

```diff
@@ -4,19 +4,17 @@ MAINTAINER Austin Riba <ariba@lco.global>
 EXPOSE 80
 ENV NODE_ENV production
 WORKDIR /valhalla
-CMD uwsgi --ini /etc/uwsgi.ini
+CMD gunicorn valhalla.wsgi -b 0.0.0.0:80
```

Comment:
Should we run more than one single worker by default? (`--workers` command line option.)
Should we send the access logs to stdout by default? (`--access-logfile=-` command line option.)

Both can easily be overridden when running this in production, so not a big deal. But it's just one more thing to remember to configure, so thought I'd point it out.

Maybe also consider running as a non-root user, and therefore use a non-privileged port number (8000 or 8080 or ???).

Reply:
Set workers to 4. We have a request logging middleware so we don't actually want access logs. I don't see the benefit of running as another user. Security? Seems like the "Docker way" is to run as a single user.